app/controllers/application_controller.rb

```diff
@@ -10,8 +10,17 @@ class ApplicationController < ActionController::Base
   # Add in params to allow things into Devise w/ Blacklight
   def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:account_update) do |user_params|
-      user_params.permit(:nickname, :fav_pokemon, :quote, :email, :password, :password_confirmation, :current_password)
+      user_params.permit(:email, :nickname, :fav_pokemon, :quote, :password, :password_confirmation, :current_password)
     end
   end
 
+  # Redirect after login
+  def after_sign_in_path_for(resource)
+    stored_location_for(pokemons) || pokemon_main_path
```

Comment:
Im not sure this route will work. Is `pokemons` variable defined anywhere? Also, if a variable in a method is unused, like resource in this case, you want to add an underscore in front of it. This denotes the fact that the variable isn't used. `_resource`

Reply:
Okay I will do that, the ```pokemons``` variable I thought they meant it as the resources variable which is in the ```routes.rb``` which have a line said ```resources: pokemons``` and that's why I used it in the scenario.

Reply:
I was following this tutorial link from Devise GitHub: https://github.com/heartcombo/devise/wiki/How-To:-Redirect-to-a-specific-page-on-successful-sign-in